conjure-java-core/src/integrationInput/java/test/prefix/com/palantir/product/Union.java

```diff
@@ -305,7 +305,6 @@ private int getValue() {
         }
 
         @Override
-        @SuppressWarnings("deprecation")
```

Comment:
Is this correct? I would have though a non-deprecated method (accept) calling a deprecated method (Visitor::visitBar) would be problematic?

Reply:
Hypothesis: if the deprecated method is defined in the same file, then the usage does not produce a warning.

Reply:
iirc we introduced the suppressions because we use javac `-Xlint:deprecation` in some places where it may apply to generated code. I'd like to ensure we're not regressing that functionality

Reply:
Okay, then would you be willing to approve something similar to https://github.com/palantir/gradle-conjure/pull/951? I'm thinking of suppressing all warnings from the generated project.

Reply:
I think that’s reasonable given the warnings may not necessarily align with deprecation checks elsewhere.